src/julia_internal.h

```diff
@@ -509,6 +509,16 @@ extern const int jl_tls_elf_support;
 void jl_init_threading(void);
 void jl_start_threads(void);
 
+// -- staticdata.c -- //
+extern char     *jl_sysimg_gvars_base;
```

Comment:
Should we be putting these into a struct rather than dlsym'ing them individually?